data/en/listtrim.json

```diff
@@ -4,14 +4,14 @@
 	"syntax": "listTrim(list [, delimiters])",
 	"returns": "string",
 	"related": [],
-	"description": "",
+	"description": "trims leading and trailing spaces from each list element",
 	"params": [
-		{"name": "list", "description": "", "required": true, "default": "", "type": "string", "values": []},
-		{"name": "delimiters", "description": "", "required": false, "default": ",", "type": "string", "values": []}
+		{"name": "list", "description": "a string list", "required": true, "default": "", "type": "string", "values": []},
+		{"name": "delimiters", "description": "characters that separate list elements. The default value is comma.", "required": false, "default": ",", "type": "string", "values": []}
```

Comment:
You don't need the second sentence ("The default value is comma") if you specified the default value in the structure.